workspaces/copilot/plugins/copilot-common/src/index.ts

```diff
@@ -23,6 +23,10 @@
 /**
  * Represents metrics data for a day in the copilot plugin.
  *
+ * This used to be part of an API exposed by GitHub (/usage),
+ * that was deprecated. At some point we should fully convert to the new CopilotMetrics way.
+ * Until the frontend is fully converted, we need to keep this type, since the new metrics converts its result to this type.
+ *
  * @public
```

Comment:
```suggestion
 * @deprecated
```

Let's mark it as deprecated so we can signal the intent to remove it at a future point more easily 👍